features/environment.py

```diff
@@ -34,19 +36,22 @@ class UAClientBehaveConfig:
     # the test framework
     boolean_options = ["image_clean", "destroy_instances"]
     str_options = ["reuse_image"]
```

Comment:
You probably want to continue to include "contract_token" in the str_options list as well because it is a str type attribute an we may find it confusing in the future about why it is excluded from this list if we are operating specially on all self.str_options. 

I realize at the moment we are only iterating over self.boolean_options in from_environ() to set a default boolean value to True, but that could change in the future and 'future us' would likely miss including both str_options and redact_options if we are operating on all string type attributes to set defaults